docs/language/overview.md

```diff
@@ -1054,17 +1085,17 @@ Record literals have the form
 where a `<spec>` has one of three forms:
 ```
 <field> : <expr>
-<ref>
+<field>
 ...<expr>
 ```
 The first form is a customary colon-separated field and value similar to JavaScript,
 where `<field>` may be an identifier or quoted string.
-The second form is an implied field reference `<ref>`, which is shorthand
-for `<ref>:<ref>`.  The third form is the `...` spread operator which expects
-a record value as the result of `<expr>` and inserts all of the fields from
-the resulting record.
+The second form is an [implied field reference](#24-implied-field-references)
+`<ref>`, which is shorthand for `<field>:<ref>`.  The third form is the `...`
```

Comment:
@nwt: Ok, I'm putting it back to the way it was originally then. That is, not only am I accepting your most recent suggestion but also changing it back to `<ref>` in the table right above. My reason for making any change was because I couldn't understand what it was trying to express as originally written, but once I realized there was a section to link to with more info on implied field references, that alone helps.